src/client/Microsoft.Identity.Client/Platforms/netcore/NetCorePlatformProxy.cs

```diff
@@ -201,6 +201,11 @@ public override Task StartDefaultOsBrowserAsync(string url)
                                 RedirectStandardOutput = true,
                                 RedirectStandardError = true
                             };
+                            string sudoUser = Environment.GetEnvironmentVariable("SUDO_USER");
+                            if (!string.IsNullOrWhiteSpace(sudoUser))

```

Comment:
Is it possible to launch the browser as a non sudo user here instead?

Reply:
I tried that by supplying name to the process. Did not work :(. If anyone knows another way, please let me know. BTW, client accepted the exception solution.